### PR TITLE
update accounts_registered counter

### PR DESCRIPTION
**Pull request by robert-zaremba**

### CODE REVIEW

File: p3-farm/src/vault.rs [hunk 1]
```diff
@@ -107,20 +107,20 @@ impl Contract {
                 / self.staked_units
     }
 
+    /// Recomputes stake aggreagator. Must be called after ping_s!
     pub(crate) fn _recompute_stake(&mut self, v: &mut Vault) {
         let mut s = min_stake(&v.staked, &self.stake_rates);
         if !v.cheddy.is_empty() {
             s += s * u128::from(self.cheddar_nft_boost) / BASIS_P;
         }
         if s > v.min_stake {
```
Comment (YellingOilbird) [inline p3-farm/src/vault.rs:116]: should we thinking about case where `s=v.min_stake`?

Comment (robert-zaremba) [reply p3-farm/src/vault.rs:116]: Why? if `s==v.min_stake` then there is nothing to update.

Comment (YellingOilbird) [reply p3-farm/src/vault.rs:116]: Yep, agree. Just thinking about it as “coverage” way. We don’t need to check this actually

**Review by YellingOilbird — APPROVED** [review]
LGTM

**Review by YellingOilbird — APPROVED** [review]

**Pull request merged**